src/charm.py

```diff
@@ -218,6 +238,24 @@ def server_cert_path(self) -> Optional[str]:
     # === UTILITY METHODS === #
     ###########################
 
+    def _update_mimir_cluster(self):  # common exit hook
+        # TODO: does this prevent disabling log forwarding on relation-broken ???
```

Comment:
this is probably something we should check before we merge

Reply:
Answered in another conversation, but this doesn't happen anymore since the worker autonomously disables log forwarding on `mimir-cluster-relation-broken`